examples/design-system/package.json

```diff
@@ -8,7 +8,8 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "changeset": "changeset",
     "version-packages": "changeset version",
-    "release": "turbo run build --filter=docs^... && changeset publish"
+    "release": "turbo run build --filter=docs^... && changeset publish",
+    "preview-storybook": "turbo preview-storybook"
```

Comment:
maybe it's not related to this PR (as in, maybe it's already happening before this PR) but this 404's for me.  does it work for you?

Reply:
@dimitropoulos If you see fit, I could add this script in another merge request too. About the 404, you have to run the build script before the preview-storybook. Let me know if there was another thing going on.